learn/install/mac.md

```diff
@@ -0,0 +1,98 @@
+Installing Python on a Mac computer is a straightforward and simple process that can be accomplished in just a few minutes. 
+  
+## How to Install Python on Mac OS  
+
+To install Python on a Mac OS, you can follow these simple steps:
+
+1. Go to the official Python website and download the latest version of Python for Mac.
+2. Once the download is complete, open the installer and follow the instructions on the screen.
+3. Once the installation process is complete, you can check Python version on your mac by opening the terminal app and typing `python3 --version`.
+4. If Python is installed successfully, the version number will be displayed in the Terminal.
+
+By following these steps, you can easily install Python on macbook and start working with this popular programming language. Remember to download Python for Mac from the official website to ensure that you have the latest and most secure version of the software.  
+
+### How to Install Python with Brew  
+
+To install Python on macOS using brew, follow these simple steps:
+
+1. Open the terminal on your macOS.
+2. Install brew if you haven't already by running the command:
+
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+3. After installation, run the command:
+
+```shell
+brew install python
+```
+
+4. The latest version of Python will be installed on your macOS.
+
+Installing Python with brew on macOS is a simple and effective way to get the latest version of Python on your system. This process will save you time and python command not found mac. 
```

Comment:
> This process will save you time and python command not found mac. 

What? o_O